lib/app/modules/settings/views/settings_view.dart

```diff
@@ -96,6 +99,15 @@ class SettingsView extends GetView<SettingsController> {
                   width: width,
                   themeController: themeController,
                 ),
+                const SizedBox(
+                  height: 20,
+                ),
+                 AboutSection(
```

Comment:
Instead of doing this, just make about as a different page using get_cli and navigate to it via GetX.